SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -19,6 +19,8 @@ public partial class Crashes : IAppCenterService
         /// </summary>
         public bool InstanceEnabled { get; set; }
 
+        private static ErrorReport LastSessionErrorReport = null;
```

Comment:
State must not be static, please look at existing code like in Analytics. And will probably involve some locking once the main task is merged.